packages/components/.stylelintrc.js

```diff
@@ -0,0 +1,150 @@
+    // 'stylelint-config-prettier-scss',
+  ],
+  rules: {
+    // =============================
+    // PRETTIER OVERRIDES:
+    // =============================
+    // 'indentation': null, // to test with prettier
+    // "indentation": 2,
+    // "string-quotes": "double",
+
+    // =============================
+    // TO FIX:
+    // =============================
+
+    // see: https://stylelint.io/user-guide/rules/list/color-hex-case/
+    'color-hex-case': 'lower',
+
+    // see: https://stylelint.io/user-guide/rules/list/declaration-colon-space-after/
+    'declaration-colon-space-after': 'always',
+
+    // see: https://stylelint.io/user-guide/rules/list/length-zero-no-unit/
+    'length-zero-no-unit': true,
+
+    // see: https://stylelint.io/user-guide/rules/list/no-eol-whitespace/
+    'no-eol-whitespace': true,
+
+    // see: https://stylelint.io/user-guide/rules/list/no-extra-semicolons/
+    'no-extra-semicolons': true,
+
+    // see: https://stylelint.io/user-guide/rules/list/no-missing-end-of-source-newline/
+    'no-missing-end-of-source-newline': true,
+
+    // see: https://stylelint.io/user-guide/rules/list/property-no-vendor-prefix/
+    'property-no-vendor-prefix': true,
+
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-empty-line-before/README.md
+    // 'scss/dollar-variable-empty-line-before': [use the default here],
+
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/double-slash-comment-whitespace-inside/README.md
+    'scss/double-slash-comment-whitespace-inside': 'always',
+
+    // see: https://stylelint.io/user-guide/rules/list/selector-combinator-space-after/
+    'selector-combinator-space-after': 'always',
+
+    // =============================
+    // TO DISCUSS:
+    // =============================
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://stylelint.io/user-guide/rules/list/alpha-value-notation/
+    'alpha-value-notation': null,
+
+    // not sure what to do with this rule, seems the default is "never" with some exceptions
+    // probably I would keep the default provided by 'stylelint-config-standard-scss'
+    // see: https://stylelint.io/user-guide/rules/list/at-rule-empty-line-before/
+    // 'at-rule-empty-line-before': null,
+
+    // preference for 'legacy'
+    // see: https://stylelint.io/user-guide/rules/list/color-function-notation/
+    'color-function-notation': null,
+
+    // would disable this rule, impacts a "grid-template-areas" definition, not sure why
+    // see: https://stylelint.io/user-guide/rules/list/declaration-block-no-redundant-longhand-properties/
+    'declaration-block-no-redundant-longhand-properties': null,
+
+    // what option here? 'never-single-line' or 'never'?
+    // see: https://stylelint.io/user-guide/rules/list/function-parentheses-space-inside/
+    'function-parentheses-space-inside': 'never-single-line',
```

Comment:
I generally don't put space inside parenthesis except when it seems it would make the code clearer. I don't have a strong preference myself so whatever others prefer is fine.